src/test/java/com/newbarams/ajaja/module/plan/presentation/PlanControllerTest.java

```diff
@@ -519,20 +517,7 @@ void createPlan_Success() throws Exception {
 			.content(objectMapper.writeValueAsString(request)));
 
 		// then
-		result.andExpectAll(
-			status().isCreated(),
-			jsonPath("$.success").value(Boolean.TRUE),
-			jsonPath("$.data.id").value(response.getId()),
-			jsonPath("$.data.userId").value(response.getUserId()),
-			jsonPath("$.data.title").value(response.getTitle()),
-			jsonPath("$.data.description").value(response.getDescription()),
-			jsonPath("$.data.iconNumber").value(response.getIconNumber()),
-			jsonPath("$.data.public").value(response.isPublic()),
-			jsonPath("$.data.canRemind").value(response.isCanRemind()),
-			jsonPath("$.data.canAjaja").value(response.isCanAjaja()),
-			jsonPath("$.data.ajajas").value(response.getAjajas()),
-			jsonPath("$.data.tags[0]").value(response.getTags().get(0))
-		);
+		result.andExpect(status().isCreated());
```

Comment:
created니까 예상한대로 경로가 출력되는지 테스트해볼까요?
예시로 들면 다음과 같습니다!
```suggestion
		result.andExpectAll(
			status().isCreated(),
			header().string(HttpHeaders.LOCATION, expectedPath)
		);
```

Reply:
수정했습니다!!